addons/website_sale/views/templates.xml

```diff
@@ -1735,6 +1735,7 @@
             - _cta_classes: CSS classes to append on the primary navigation button; default `None`.
             - _form_send_navigation: Whether the primary button serves as a submit button for a
                                      form; default `None`.
+            - is_error: Whether there is not possible to proceed the payment; default `None`.
```

Comment:
hide_payment_button: Whether the payment button should be hidden; default: `False`.